staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -133,23 +133,30 @@ func ObjectReaction(tracker ObjectTracker) ReactionFunc {
 			return true, nil, nil
 
 		case PatchActionImpl:
-			obj, err := tracker.Get(gvr, ns, action.GetName())
```

Comment:
This patch increases the complexity of the code. Can we merely test for `ApplyPatchType` and fail appropriately right at the beginning?

Reply:
Fixed, thanks